openapi.yaml

```diff
@@ -51,6 +54,63 @@ info:
   version: 0.1.4
 openapi: 3.0.2
 paths:
+  /federated/ingest_metadata:
+    post:
+      description: Decrypt payload, process metadata, file secret and send success
+        event
+      operationId: ingestFileUploadMetadata
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/EncryptedPayload'
+        required: true
+      responses:
+        '202':
+          content:
+            application/json:
+              schema: {}
+          description: Received and decrypted data successfully.
+        '422':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HTTPValidationError'
+          description: Validation Error
+      security:
+      - HTTPBearer: []
+      summary: Processes encrypted output data from the S3 upload script and ingests
+        it into the Encryption Key Store, Internal File Registry and Download Controller.
+      tags:
+      - FileIngestService
+  /federated/ingest_secret:
+    post:
+      description: Decrypt payload, process metadata, file secret and send success
```

Comment:
```suggestion
      description: Decrypt payload and deposit file secret in exchange for a secret id
```

Reply:
Changed in 50ac60480d47978cbffb98758dd259f95c18bc3d